### PR TITLE
IDP-790 part 1 -- add php-cs-fixer

### PR DESCRIPTION
**Pull request by briskt**
### Added
- Added php-cs-fixer to dev dependencies and Makefile

[IDP-790](https://itse.youtrack.cloud/issue/IDP-790)

### CODE REVIEW

File: application/versions.txt [hunk 1]
```diff
@@ -7,7 +7,8 @@ codeception/module-yii2                1.1.8              Codeception module for
 codeception/specify                    2.0.0              BDD code blocks for PHPUnit...
 codeception/verify                     2.3.0              BDD assertion library for P...
 fillup/fake-bower-assets               2.0.9              Use Composer "replace" to f...
```
Comment (mtompset) [inline application/versions.txt:9]: As an aside, perhaps something other than fillup/fake-bower-assets would be good to look for in another PR.

**Review by mtompset — APPROVED** [review]

File: application/versions.txt (hunk 1 above)
Comment (forevermatt) [reply application/versions.txt:9]: Agreed (though low priority). I believe "asset-packagist" replaces the ned for that, now, fwiw.
